### DISCUSSION TITLE
code fails in pipeline

### DISCUSSION THREAD

**Discussion by amena009** [DevSkillsHQ/discusssions#41] [Help] [answered]
Hello,
I have pushed my code to implementation branch using .net and angular boilerplate. Before pushing my code, i have run and build the solution properly and also test cases. Everything is fine locally but it fails in the pipeline. See attached image for reference. Kindly help asap. (
<img width="927" alt="Capture" src="https://github.com/DevSkillsHQ/discusssions/assets/65390640/8d535f93-8cc1-4a63-80c3-7658ffec7992">
need to submit test)

**Comment by ludvig** [comment]
Hello @amena009 

There is an issue with the boilerplate (`fullstack-boilerplate-dotnet-angular`) where some `package.json` scripts to run the service were not included. This is fixed now but unfortunately not applied to your repository automatically.

To fix it in your GitHub repository please manually copy the `scripts` section from the boilerplate https://github.com/DevSkillsHQ/fullstack-boilerplate-dotnet-angular/blob/main/package.json#L5-L11 and replace the `scripts` section in your `package.json` file located in the root of your challenge GitHub repository.

Sorry for the inconvenience and let me know if you run into any issues!

**Comment by amena009** [comment]
thank you for your prompt reply. But now I am facing issue with test cases. Please suggest some solution.
<img width="958" alt="Capture" src="https://github.com/DevSkillsHQ/discusssions/assets/65390640/82e07306-2997-48d8-bdf8-f78dcc189922">

**Reply by ludvig** [reply]
Hi!

There seems to be another config mismatch for this boilerplate. In your GitHub repository there is a file called `cypress.config.json` you can replace it with the following contents. Basically we are adding the missing `baseUrl` which is used by the tests to connect to your frontend. Sorry for the inconvenience, we will be working on improving this boilerplate experience.

``` 
const { defineConfig } = require('cypress')

module.exports = defineConfig({
  env: {
    apiUrl: 'http://localhost:5000',
  },
  e2e: {
    setupNodeEvents(on, config) {},
    supportFile: false,
    baseUrl: 'http://localhost:4200',
  },
})
```

**Reply by amena009** [reply]
still facing issues with test cases :(

**Reply by ludvig** [reply]
Hi, in your GitHub repository please modify the file `app-frontend/package.json` and replace the `start` script with:

``` 
"start": "ng serve --host=127.0.0.1",
```

I believe there is some issue with node 18 and dns lookup on GitHub actions.

**Reply by amena009** [reply]
tried updating app-frontend/package.json as you told but the now facing the below error.

<img width="953" alt="Capture" src="https://github.com/DevSkillsHQ/discusssions/assets/65390640/adcdaafb-0739-4221-a193-bf0f5cd1240f">

**Reply by ludvig** [reply, marked as answer]
Hi, it seems you missed the opening quotation mark `"` when you copied the code suggestion, copy exactly the content:
```
"start": "ng serve --host=127.0.0.1",
```
